src/app/(admin)/admin/knowledge/components/KnowledgeEditor.tsx

```
@@ -181,51 +422,179 @@ export function KnowledgeEditor({ entry, onSave, onCancel }: KnowledgeEditorProp
+            {/* Quick add common metadata fields */}
+            {Object.entries(metadataFieldTypes).map(([fieldKey, config]) => {
+              if (!formData.metadata.hasOwnProperty(fieldKey)) {
+                return (
+                  <button
+                    key={fieldKey}
+                    type="button"
+                    onClick={() => {
+                      const defaultValue = config.type === 'tags' ? [] : 
+                                         config.type === 'date' ? new Date().toISOString() :
+                                         config.type === 'select' && config.options ? config.options[0] : '';
+                      handleMetadataChange(fieldKey, defaultValue);
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)